docs/source/guides/writer/chapters/recipes.rst

```diff
@@ -0,0 +1,59 @@
+Defining what to run using recipe files
+---------------------------------------
+
+If you've followed the previous documentation sections, you should now
+be able to write ``exec-test`` tests and also ``avocado-instrumented``
+tests.  These tests you should be found when you run
+``avocado run /reference/to/a/test``.  Internally, though, these will
+be describe as a :class:`avocado.core.nrunner.runnable.Runnable`.
```

Comment:
typo: describe/described

Reply:
I actually meant "defined".  Changing it here, thanks!